### DISCUSSION TITLE
Using composer package in a module

### DISCUSSION THREAD

**Discussion by rickybulalakaw** [nWidart/laravel-modules#1964] [Q&A]
Hi! 

I installed livewire-powergrid in my module. This package facilitates creating datatables based on Livewire easier and outputs a Livewire component class by using php artisan powergrid:create. 

Is there a proper artisan command to use this package within a module? When I use it, the output file goes to the main Livewire folder of the app and I have to manually transfer it to the Livewire folder in the module (I installed modules-livewire) to change the namespaces. 

Appreciate this package! Thanks!

**Comment by dcblogdev** [comment]
I think you will have to continue to move the files yourself that package is probaby setup to use the normal Laravel paths.
